<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GetWeightWindow</class>
 <widget class="QDialog" name="GetWeightWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>184</width>
    <height>100</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>184</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Введите данные</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QDateEdit" name="getDate">
     <property name="cursor">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="currentSection">
      <enum>QDateTimeEdit::MonthSection</enum>
     </property>
     <property name="displayFormat">
      <string>dd-MM-yyyy</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="editWeight">
     <property name="cursor">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="suffix">
      <string> кг</string>
     </property>
     <property name="maximum">
      <double>999.990000000000009</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="btnYesNo">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
